### PR TITLE
PIL-214 PIL-215 PIL-216 Fix/fix jest/eslint/flow in pure-rn branch

### PR DESCRIPTION
**Pull request by alazarevski**
Added missing modules for jest tests
Updated screenshots
Skipped Assets.test.js (TODO: update config to run this screen test)

Removed Reactotron completely from configStore Added missing test libs enzyme and the adapter for enzyme Added image extension fix for flow Moved ignored modules to [untyped] modules to solve the module resolution in flow

Fixes eslint to run the pure flow and eslint without errors

### CODE REVIEW

File: old.android/rn-libs/react-native-image-capinsets/ImageCapInset.android.js [hunk 1]
```diff
@@ -21,10 +21,12 @@ class ImageCapInset extends Component {
     return (
       <View {...rest}>
         <RCTImageCapInset
-          style={{position: 'absolute', top: 0, left: 0, bottom: 0, right: 0}}
+          style={{
+ position: 'absolute', top: 0, left: 0, bottom: 0, right: 0,
+}}
```
Comment (poocart) [inline old.android/rn-libs/react-native-image-capinsets/ImageCapInset.android.js]: Please fix spacing here.

File: src/screens/Asset/__tests__/Asset.test.js [hunk 2]
```diff
@@ -80,7 +80,7 @@ describe('Asset', () => {
     },
   };
 
-  it('renders the Asset correctly', () => {
+  xit('renders the Asset correctly', () => {
```
Comment (poocart) [inline src/screens/Asset/__tests__/Asset.test.js]: `xit` means to skip the test, why we added it here?

Comment (poocart) [inline src/screens/Asset/__tests__/Asset.test.js]: `xit` means to skip the test, why we added it here?

**Comment by poocart** [comment]
Let's get rid of changes in `old.android` as the purpose if it is to migrate it step by step not making changes to old files itself.

File: src/screens/Asset/__tests__/Asset.test.js (hunk 2 above)
Comment (alazarevski) [reply src/screens/Asset/__tests__/Asset.test.js]: now it will work with 3faa5cf

**Comment by poocart** [comment]
Why we added `stacktrace-parser` which wasn't here before? For me this looks like hack as if it's removed then tests are failing due timeouts which means we need to fix the tests and not hack them just to make it work, please correct me if I'm wrong here. Same applies for additional patch of `react-native+0.59.10.patch`.

**Comment by poocart** [comment]
As well please revert deletion of `patches/asyncstorage-down+4.2.0.patch` or move patches to `old.patches`. My assumption we will need it later as `asyncstorage-down` seems to be required by `react-native-level-fs` which is part of shimming process.

**Pull request merged**